docs/EN_US/ContainerizedHPCC/ContainerizedMods/LocalDeployment.xml

```diff
@@ -163,31 +185,287 @@ sasha-wu-archiver     ClusterIP    10.111.34.240   &lt;none&gt;       8877/TCP
+        </emphasis>option specifies the base directory:</para>
+
+        <para>The path <emphasis
+        role="bold">/run/desktop/mnt/host/c/hpccdata</emphasis> provides
+        access to the host file system for WSL2.</para>
+
+        <para>The path <emphasis role="bold">/c/hpccdata</emphasis> provides
+        access to the host file system for Hyper-V.</para>
+
+        <para>The path <emphasis role="bold">/Users/myUser/hpccdata</emphasis>
+        provides access to the host file system for Mac OSX.</para>
+
+        <para>The path <emphasis role="bold">~/hpccdata</emphasis> provides
+        access to the host file system for Linux.</para>
+
+        <variablelist>
+          <varlistentry>
+            <term>Note:</term>
+
+            <listitem>
+              <para>The value passed to --set common-hostpath is case
+              sensitive.</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </listitem>
+
+      <listitem>
+        <para>Copy the output from the helm install command in the previous
+        step from the word <emphasis role="bold">storage:</emphasis> to the
+        end, and save it to a text file.</para>
+
+        <para>In this example, we will call the file mystorage.yaml. The file
+        should look similar to this:</para>
+
+        <para><programlisting>storage:
+  planes:
+  - name: dali
+    pvc: dali-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/dalistorage"
+    category: dali
+  - name: dll
+    pvc: dll-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/queries"
+    category: dll
+  - name: sasha
+    pvc: sasha-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/sasha"
+    category: sasha
+  - name: debug
+    pvc: debug-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/debug"
+    category: debug
+  - name: data
+    pvc: data-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/hpcc-data"
+    category: data
+  - name: mydropzone
+    pvc: mydropzone-hpcc-localfile-pvc
+    prefix: "/var/lib/HPCCSystems/dropzone"
+    category: lz
+
+sasha:
+  wu-archiver:
+    plane: sasha
+  dfuwu-archiver:
+    plane: sasha</programlisting></para>
+      </listitem>
+
+      <listitem>
+        <para>If you are using Docker Desktop with Hyper-V, add the shared
+        data folder (in this example, C:\hpccdata) in Docker Desktop's
+        settings by pressing the Add button and typing c:\hpccdata.</para>
+
+        <para>This is <emphasis role="bold">not</emphasis> needed in a MacOS
+        or WSL 2 environment.</para>
+
+        <graphic fileref="../../images/dockerFileShare.jpg" />
+      </listitem>
+
+      <listitem>
+        <para>Finally, install the hpcc Helm chart, and provide a yaml file
+        that provides the storage information created by the previous
+        step.</para>
+
+        <programlisting>helm install mycluster hpcc/hpcc --version-f mystorage.yaml --version=8.6.14</programlisting>
```

Comment:
> --version-f 

looks like an extra -"-version" has slipped in here ahead of the -f. Line should be:

helm install mycluster hpcc/hpcc --version=8.6.14 -f mystorage.yaml 

it doesn't really matter, but I'd put --version ahead of -f in general, because it is quite typical to pass in multiple -f